convenor/controller/contEditAssessment.php

```diff
@@ -0,0 +1,124 @@
+<?php  
+	//Prevents scripting and SQL injection
+    function test_input($data) {
+        $data = trim($data);
+        $data = stripslashes($data);
+        $data = htmlspecialchars($data);
+        return $data;
+    }
+
+    function validate_input(&$assTitle, &$unitID, &$semester, &$year, &$description, &$isIndividualGroup, &$dueDate, &$markingGuidePath, &$proName) {
+
+        //Check Assignment Title isn't empty
+        if (empty($_POST['ass_Title'])){
```

Comment:
`ass_Title` I love it

Reply:
Thank you. I was just following the projects naming conventions - all fields should start with `ass` when possible